src/components/Landing/QAs.tsx

```diff
@@ -19,20 +19,32 @@ export default function QAs() {
     {
       q: t`What does Juicebox cost?`,
       a: [
-        t`Juicebox is an open protocol on Ethereum that is funded using Juicebox itself. You can check out the contractualized budget specs at https://juicebox.money/#/p/juicebox.`,
+        <Trans>
+          Juicebox is an open protocol on Ethereum that is funded using Juicebox
+          itself. You can check out the contractualized budget specs{' '}
+          <a
+            href="https://juicebox.money/#/p/juicebox"
+            rel="noreferrer"
+            target="_blank"
+          >
+            {' '}
+            here
+          </a>
+          .
+        </Trans>,
         t`5% of all money made by projects is sent to help pay for the maintenance and development of Juicebox itself. In exchange, projects get Juicebox tokens ($JBX), which will be worth more as the ecosystem grows over time.`,
       ],
     },
     {
       q: t`What is overflow?`,
       a: [
-        t`If you know how much your project needs to earn over some period of time to be sustainable, you can set a funding target with that amount. If your project earns more than that, the surplus funds are locked in an overflow pool. Anyone holding your project's tokens can claim a portion of the overflow pool in exchange for burning their tokens.`,
+        t`If you know how much your project needs to earn over some period of time to be sustainable, you can set a funding target with that amount. If your project earns more than that, the surplus funds are locked in an overflow pool. Anyone holding your project's tokens can claim a portion of the overflow pool in exchange for redeeming their tokens.`,
```

Comment:
I think we should reserve the work 'burning' for when a project has no overflow, tokens are redeemed for no ETH in return?

Reply:
same page